integration-tests/suites/common/utils.go

```diff
@@ -24,3 +24,12 @@ func QuoteArgs(args []string) []string {
 	}
 	return quotedArgs
 }
+
+// Returns the min of two integers.
+// Strangely there is no such built in or function in math
```

Comment:
Actually, min and max were added to the 1.21 version of golang: https://tip.golang.org/doc/go1.21

I believe they didn't exist because of the lack of templates until that version. Could we bump the required version for our tests and use the builtin instead?

Reply:
I think we should probably try to use the version of Go used by stackrox/stackrox, which is currently 1.20.10. That is the version that I have locally. Since the tests are containerized, I could try just running them that way with different versions of Go on my host and the container. However, that is currently not working for me. I will merge as is and try to come back to this issue.